jdk/make/CopyFiles.gmk

```diff
@@ -23,6 +23,23 @@
 # questions.
 #
 
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018, 2018 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
```

Comment:
The license doesn't need to be duplicated.

Reply:
Removed duplicated license.